pages/reference/api/transcode-video-storj.en-US.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Transcode Video with Storj
+---
+
+## Introduction
+
+This guide will describe how to use the Livepeer [transcode API](/reference/api/transcode-video) to transcode and store video files in Storj using Storj’s S3 API.
+
+There are [code examples](#code-examples) provided for reference.
+
+## Storj S3 Credentials
+
+This step assumes that you have already created an account at [https://www.storj.io/](https://www.storj.io/).
+
+Follow the [Storj guide for generating S3 credentials](https://docs.storj.io/dcs/api-reference/s3-compatible-gateway#I7p7Q) using either the web interface or `uplink`. Make sure that the credentials have the proper read/write permissions for the bucket that your video files will be read from and that transcoded videos will be written to. The access key and secret key of the credentials will be used in the next step.
+
+## Upload Video to Storj
```

Comment:
can we link to Storj's upload docs here?

Reply:
> can we link to Storj's upload docs here?

I don't think there is a good link to Storj documentation. We could like this one: https://docs.storj.io/dcs/getting-started/quickstart-uplink-cli/uploading-your-first-object.

But it's using `uplink` and there are two other (simpler) methods for file upload (web interface and AWS S3 CLI).